src/main/java/at/tugraz/sw20asd/lang/ui/Controller/StudyVocab.java

```diff
@@ -0,0 +1,226 @@
+            {
+                String new_answer = given_label_list.get(counter).getText();
+                String new_given = answer_label_list.get(counter).getText();
+                String new_given_language = study_label.getText();
+                String new_study_language = given_label.getText();
+
+                given_label_list.get(counter).setText(new_given);
+                answer_label_list.get(counter).setText(new_answer);
+                given_label.setText(new_given_language);
+                study_label.setText(new_study_language);
+            }
+        }
+
+        private void populate(){
+            user_info.setVisible(false);
+            show_all_btn.setVisible(true);
+            switch_btn.setVisible(true);
+            given_label.setVisible(true);
+            study_label.setVisible(true);
+            anchor_pane.getChildren().remove(choice_label);
+            anchor_pane.getChildren().remove(language_choice);
+            anchor_pane.getChildren().remove(start_btn);
+
+            int min_height = 30;
+            int margin = 10;
+            for (int counter = 0; counter < words.size(); counter++) {
+
+                String given_str;
+                String answer_str;
+                if(study_language.equals("de")){
+                    given_str = words.get(counter).getTranslation();
+                    answer_str = words.get(counter).getPhrase();
+                }
+                else{
+                    given_str = words.get(counter).getPhrase();
+                    answer_str = words.get(counter).getTranslation();
+                }
+
+                Label given = new Label(given_str);
+                given.setId("given" + given_label_list.size());
+                given.setMinHeight(min_height);
+                setMargin(given, new Insets(10,0,0,0));
+                given_label_list.add(given);
+                given_list.getChildren().add(given);
+
+                Label answer = new Label(answer_str);
+                answer.setId("answer" + answer_label_list.size());
+                answer.setVisible(false);
+                answer.setMinHeight(min_height);
+                setMargin(answer, new Insets(10,0,0,0));
+                answer_label_list.add(answer);
+                answer_list.getChildren().add(answer);
+
+                Button show_answer = new Button();
+                show_answer.setText("Show Answer");
+                show_answer.setId("btn" + answer_button_list.size());
+                show_answer.setMinHeight(min_height);
+                setMargin(show_answer, new Insets(10,0,0,0));
+                show_answer.setOnAction(new EventHandler<ActionEvent>() {
+                    public void handle(ActionEvent event) {
+                        if(show_answer.getText().equals("Show Answer")){
+                            answer.setVisible(true);
+                            show_answer.setText("Hide Answer");
```

Comment:
Please, consider using an underlying model and binding logic, such as, showing text to its state instead of controlling UI directly.